docs/guides/usage/tracing.livemd

```diff
@@ -83,7 +120,7 @@ Integer.parse("3")
 <!-- livebook:{"output":true} -->
 
 ```
-Matcha.Trace: `Elixir.Integer.parse("1")` called on #PID<0.142.0>
```

Comment:
This pid will change every time, so if the content of the pid changes between runs of the example, I'd rather ignore those diffs.